Public/Get-JenkinsUserFullName.ps1

```diff
@@ -0,0 +1,21 @@
+function Get-JenkinsUserFullName {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$true)]
+        [string] $Username,
+        [Parameter(Mandatory=$true)]
+        [securestring] $Password
+    )
+
+    if (!$script:fullNamesByUser) {
+        $script:fullNamesByUser = [hashtable]::new()
+    }
+
+    if ($fullName = $fullNamesByUser[$Username]) {
+        return $fullName
+    } else {
+        $response = Invoke-JenkinsRequest -Resource "/user/me/api/json"
```

Comment:
I'd prefer a method for `Get-CurrentJenkinsUser` `Invoke-JenkinsRequest -Resource "/user/me/api/json"` and this method just calling that.

Reply:
Done. Splitted up into cmdlets `Get-JenkinsUserInfo` and `Get-JenkinsUserFullName`.